packages/babel-traverse/src/path/introspection.js

```diff
@@ -109,6 +109,24 @@ export function canSwapBetweenExpressionAndStatement(replacement) {
   return false;
 }
 
+/**
+ * Check whether the current path contains an `await` expression
+ */
+
+export function containsAwaitExpression(): boolean {
```

Comment:
You can/should use `hasType` instead:

https://github.com/babel/babel/blob/f92c2ae830dbb32013a36fa74facd2ef95b9947d/packages/babel-plugin-transform-block-scoping/src/index.js#L572-L576

Reply:
Disclaimer: I didn't know about that method until 5 mins ago. Usually, when someone adds a new method to `@babel/traverse` there is a 50% probability that it already exists.

Reply:
I tried to look for it too... But I guess you can find things better than me